src/db.c

```diff
@@ -235,6 +236,10 @@ parse_rule(char *rule_node)
       if        (!strcmp(*rule, NODE_COMMAND)) {
         value = parse_value(rule_path, *rule);
         if (value != NULL) {
+          // handle value as case-insensitive
+          for (i = 0; i < strlen(value); i++) {
+            value[i] = tolower(value[i]);
+          }
           res->cmd = policy_parse_command_string(value);
```

Comment:
Seems like policy_parse_command_string() should just call strcasecmp().  With that, ctype.h is no longer needed.

Reply:
Fixed